src/pcapi/models/provider.py

```diff
@@ -40,3 +40,14 @@ def isAllocine(self) -> bool:
         from pcapi import local_providers  # avoid import loop
 
         return self.localClass == local_providers.AllocineStocks.__name__
+
+    @property
+    def implementsProviderAPI(self) -> bool:
```

Comment:
Puisque c'est une _property_, c'est du "snake_case". 

```suggestion
    def implements_provider_api(self) -> bool:
```